bids-validator/bids_validator/rules/file_level_rules.json

```
@@ -1,60 +1,157 @@
+  "anat_group1": {
+    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?anat[\\/\\\\]\\1(_\\2)?(?:_acq-[a-zA-Z0-9]+)?(?:_ce-[a-zA-Z0-9]+)?(?:_rec-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_part-(imag|mag|phase|real))?_(?:@@@_anat_suffixes_@@@)\\.(@@@_anat_ext_@@@)$",
     "tokens": {
       "@@@_anat_suffixes_@@@": [
         "T1w",
         "T2w",
-        "T1map",
-        "T2map",
-        "T1rho",
+        "PDw",
+        "T2starw",
         "FLAIR",
-        "PD",
-        "PDT2",
         "inplaneT1",
         "inplaneT2",
-        "angio",
-        "SWImagandphase",
-        "T2star",
-        "FLASH",
+        "PDT2",
+        "angio"
+      ],
+      "@@@_anat_ext_@@@": ["nii\\.gz", "nii", "json"]
+    }
+  },
+
+  "anat_group2": {
+    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?anat[\\/\\\\]\\1(_\\2)?(?:_acq-[a-zA-Z0-9]+)?(?:_ce-[a-zA-Z0-9]+)?(?:_rec-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?_(?:@@@_anat_suffixes_@@@)\\.(@@@_anat_ext_@@@)$",
+    "tokens": {
+      "@@@_anat_suffixes_@@@": [
+        "T1map",
+        "T2map",
+        "T2starmap",
+        "R1map",
+        "R2map",
+        "R2starmap",
         "PDmap",
-        "photo"
+        "MTRmap",
+        "MTsat",
+        "UNIT1",
+        "T1rho",
+        "MWFmap",
+        "MTVmap",
+        "PDT2map",
+        "Chimap",
+        "TB1map",
+        "RB1map",
+        "S0map",
+        "M0map"
       ],
-      "@@@_anat_ext_@@@": ["nii\\.gz", "nii", "json"],
-      "@@@_cont_ext_@@@": [
-        "_physio\\.tsv\\.gz",
-        "_stim\\.tsv\\.gz",
-        "_physio\\.json",
-        "_stim\\.json"
-      ]
+      "@@@_anat_ext_@@@": ["nii\\.gz", "nii", "json"]
     }
   },
 
-  "anat_defacemask": {
+  "anat_group3": {
     "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?anat[\\/\\\\]\\1(_\\2)?(?:_acq-[a-zA-Z0-9]+)?(?:_rec-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_mod-(?:@@@_anat_suffixes_@@@))?_defacemask\\.(@@@_anat_ext_@@@)$",
     "tokens": {
       "@@@_anat_suffixes_@@@": [
         "T1w",
         "T2w",
-        "T1map",
-        "T2map",
-        "T1rho",
+        "PDw",
+        "T2starw",
         "FLAIR",
-        "PD",
-        "PDT2",
         "inplaneT1",
         "inplaneT2",
+        "PDT2",
         "angio",
-        "SWImagandphase",
-        "T2star",
-        "FLASH",
+        "T1map",
+        "T2map",
+        "T2starmap",
+        "R1map",
+        "R2map",
+        "R2starmap",
         "PDmap",
-        "photo"
+        "MTRmap",
+        "MTsat",
+        "UNIT1",
+        "T1rho",
+        "MWFmap",
+        "MTVmap",
+        "PDT2map",
+        "Chimap",
+        "TB1map",
+        "RB1map",
+        "S0map",
+        "M0map",
+        "MESE",
+        "MEGRE",
+        "VFA",
+        "IRT1",
+        "MP2RAGE",
+        "MPM",
+        "MTS",
+        "MTR"
       ],
       "@@@_anat_ext_@@@": ["nii.gz", "nii"]
     }
   },
 
+  "anat_group4": {
```

Here I'm going to make some suggestions, but you might have better names. Or we could just go with `group1`, since they are grouping suffixes.

```suggestion
  "anat_multiecho": {
```

---

I'd like to have the groups here match the groups in the schema, but adding descriptive names to the schema sounds better than adding bad names here. I'll accept your recommendation and then open a PR to the specification with the name change there.